docs/installation.rst

```diff
@@ -121,4 +232,18 @@ You can also try running an example input file:
 
 .. code-block:: console
 
-    python -m desc -vvv examples/DESC/SOLOVEV
+    python -m desc -vv desc/examples/SOLOVEV
+
+Troubleshooting
+***************
+We list here some common problems encountered during installation and their possible solutions.
+If you encounter issues during installation, please `leave us an issue on Github <https://github.com/PlasmaControl/DESC/issues>`_ and we will try our best to help!
+
+ - **Problem**: I've installed DESC, but when I check my installation I get an error `ModuleNotFoundError: No module named 'desc'`.
+ - **Solution**: This may be caused by DESC not being on your PYTHONPATH, or your environment containing DESC not being activated.
+     - Try adding the DESC directory to your PYTHONPATH manually by adding the line `export PYTHONPATH="$PYTHONPATH:path/to/DESC"` (where `/path/to/DESC` is the path to the DESC folder on your machine) to the end of your ~/.bashrc (or other shell configuration) file.
+       You will also need to run `source ~/.bashrc` after making the change to ensure that your path updates properly for your current terminal session.
+     - Try ensuring you've activated the conda environment that DESC is in ( `conda activate desc-env` ), then retry using DESC.
+ - **Problem**: I've installed DESC, but when I check my installation I get an error `ModuleNotFoundError: No module named 'termcolor'` (or another module which is not `desc```).
+ - **Solution**: you likely are not running python from the environment in which you've installed DESC.
+     - Try ensuring you've activated the conda environment that DESC is in( `conda activate desc-env` ), then retry using DESC.
```

Comment:
The code snippets `with formatting like this` don't seem to be rendering correctly here. It might be a formatting issue with being a bullet point list.